### ISSUE TITLE
colormap not displaying right

### ISSUE THREAD

**Issue by JoannaLeng** [OpenChemistry/tomviz#2193]
I just updated my version of TomViz as I last installed it about 18 months ago. I do not know what version I had but I now have verion 1.10.0. When I run this the colormap is not rendered correctly - it is rendered as a pattern of blocks that alter when I move the mouse across it and goes across the whole graphs area of that window - an image is attached. I have tried to repair the install and I have tried to uninstall and reinstall but neither of these fix it. I have ParaView and run that to check to see if I could see problems with the colormap there to see if it is not a problem with my system - ParaView was fine.
![tomviz_colormap_rended_wrong](https://user-images.githubusercontent.com/23033397/132362501-f9d839e7-6ffd-47ca-b4b6-5f2ed5068a8a.jpg)

**Comment by JoannaLeng** [comment]
I forgot to mention that I am on a Window10 syste. I have now downloaded both of the tomviz zip files and both of them have the same issue. I forgot to say that this was an issue with my old version that I needed to update. This makes me think this is a graphics configuration issue. A few months ago I altered the python graphics backend. Is the Colormap a python component and does it need a particular backend?

**Comment by cjh1** [comment]
@JoannaLeng Thanks for reporting this. The colormap is build up in C++ so shouldn't be effected by changes you made to python. It is strange that ParaView doesn't exhibit that same behavior.

**Comment by JoannaLeng** [comment]
I noticed while I was using the tomviz versions in the zip files that there is a ParaView executable in the bin directory too. After your comment I realised I had used ParaView installed on my system not the one in the Tomiz bin directory. When I used the version of ParaView that is packaged with TomViz I got the same problem with the colormap with the veersion installed separately on my system I did not - see the attched image.
![paraview_colormap_rended_wrong](https://user-images.githubusercontent.com/23033397/132865261-0dd6d2c9-5073-4fcd-bad2-05b3202d632e.jpg)
I am running this on a Windows 10 system, with a	Intel(R) Xeon(R) CPU E3-1271 v3 @ 3.60GHz, 3601 Mhz, 4 Core(s), 8 Logical Processor(s) and using a NVidia Quadro 2000

**Comment by JoannaLeng** [comment]
I first installed TonViz on this system in March/April 2020. Is there anyway I could access the version that was released then so I can see if the problem with the colormap appears. It could help narrow down when the issue first happened. Also I am just starting a new project and having a working verion of TomViz for that would be really good :-)

**Comment by cjh1** [comment]
@JoannaLeng Absolutely you can access past releases [here](https://github.com/OpenChemistry/tomviz/releases). Narrowing down which release it was broken in would be great, I am guessing that 1.9.0 will probably work, fingers crossed.

**Comment by JoannaLeng** [comment]
Yes!!! version 1.9.0 works. Let me know if you need any further information etc.
